server.go

```diff
@@ -584,3 +585,44 @@ func runHTTPandHTTPSBandwidthHandler(w http.ResponseWriter, r *http.Request, p E
 		atomic.AddUint64(&test.testResult.data, uint64(r.ContentLength))
 	}
 }
+
+func runHTTPLatencyServer() {
+	sm := http.NewServeMux()
+	sm.HandleFunc("/", runHTTPLatencyHandler)
+	l, err := net.Listen(tcp(ipVer), ":"+httpLatencyPort)
+	if err != nil {
+		ui.printErr("Unable to start HTTP server. Error in listening on socket: %v", err)
+		return
+	}
+	ui.printMsg("Listening on " + httpLatencyPort + " for HTTP latency tests")
+	go runHTTPServer(tcpKeepAliveListener{l.(*net.TCPListener)}, sm)
+}
+
+func runHTTPLatencyHandler(w http.ResponseWriter, r *http.Request) {
+	runHTTPProtoLatencyHandler(w, r, HTTP)
+}
+
+func runHTTPProtoLatencyHandler(w http.ResponseWriter, r *http.Request, p EthrProtocol) {
```

Comment:
This function is almost same as HTTPandHTTPSBandwidthHandler, so please move this to common function as well. Something like:
runHTTPandHTTPsHandler(), so only additional code in runHTTPandHTTPsBandwidthHandler is atomic.Add...

Reply:
Done